lib/Doctrine/ORM/Tools/Console/Command/RunDqlCommand.php

```diff
@@ -101,6 +95,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         $resultSet = $query->execute([], constant($hydrationMode));
 
-        $ui->text(Debug::dump($resultSet, $input->getOption('depth'), true, false));
+        $dumper = new CliDumper(static function (string $payload) use ($output) : void {
+            $output->write($payload);
+        });
+        $dumper->dump((new VarCloner())->cloneVar($resultSet));
```

Comment:
This change specifically requires an integration test to be added

Reply:
There already is a test, which was failing without this change:
https://github.com/doctrine/doctrine2/blob/master/tests/Doctrine/Tests/ORM/Tools/Console/Command/RunDqlCommandTest.php#L67